src/write_http.c

```diff
@@ -290,7 +290,10 @@ static int wh_flush(cdtime_t timeout,
     return ENOMEM;
   }
 
-  return wh_post(cb, json);
+  int status = wh_post(cb, json);
+  free((char *)json);
```

Comment:
I'd prefer to drop the `const` keyword from the `json` variable ­– some compilers are bound to complain about disregarding the const keyword here. When done, you can remove the cast here.